src/serverlessFunctions/parseFromTemplate.js

```diff
@@ -33,7 +33,7 @@ export default (template, envVars, portOffset, basePath, refOverrides = {}) => {
       .filter(isApiEvent)
       .map((ApiEvent, i, apiEvents) => [
         name + (apiEvents.length > 1 ? `_${i}` : ''),
-        { ApiEvent, ...resource },
+        { ApiEvent, OriginalName: name, ...resource },
```

Comment:
OriginalName refers to the function name right, not the name of the event? I think we could rename it to `functionName` then, it would be a bit more obvious. What do you think @Kivol?

Reply:
Agreed. I've renamed it.